.github/renovate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "group:allNonMajor",
+    "schedule:weekly"
+  ],
+  "ignoreUnstable": true,
+  "labels": ["dependencies"],
+  "branchPrefix": "renovate/",
+  "reviewers": [
+    "sammysteiner",
+    "acouch",
+    "daphnegold"
+  ],
+  "rangeStrategy": "update-lockfile",
+  "timezone": "America/Los_Angeles",
+  "vulnerabilityAlerts": {
+    "enabled": true,
+    "labels": [
+      "vulnerability"
+    ]
+  },
+  "packageRules": [
+    {
+      "description": "Group React packages together",
+      "packageNames": [
+        "react",
+        "@types/react",
+        "react-dom",
+        "@types/react-dom"
+      ],
+      "groupName": "React"
+    },
+    {
+      "description": "Group Storybook packages together",
+      "matchPackagePatterns": ["storybook"],
+      "groupName": "Storybook"
+    },
+    {
+      "description": "Group I18next packages together",
+      "matchPackagePatterns": ["i18next"],
+      "groupName": "I18next"
+    }
```

Comment:
```suggestion
    {
        "description": "Group I18next packages together",
        "matchPackagePatterns": ["i18next"],
        "groupName": "I18next"
    },
    {
        "description": "@uswds/uswds must match the version that react-uswds depends on, so it should be manually updated.",
        "enabled": false,
        "matchPackagePatterns": ["@uswds/uswds"],
        "groupName": "USWDS core"
    },
    {
        "description": "Leave peer dependencies alone",
        "depTypeList": ["peerDependencies"],
        "enabled": false
    }
```

Reply:
In the same vein, should we add some configuration for for Python groupings? We can always do that later/track it in a follow up ticket. I am guessing we plan to use Renovate across our entire monorepo.

Reply:
Did you mean to set those `enabled` flags to `true`? I saw that in the next-template repo and decided agains those specifically since the next template doesn't have them enabled.